src/mcts/node.h

```diff
@@ -363,8 +364,9 @@ class EdgeAndNode {
   }
 
   int GetVisitsToReachU(float target_score, float numerator,
-                        float default_q) const {
-    const auto q = GetQ(default_q);
+                        float default_q, bool logit_q) const {
+    const auto q = (logit_q ? FastLogit(0.99999999 * GetQ(default_q)) :
```

Comment:
0.99999999 is a double constant and doesn't do what you think. You probably want 0.99999994f instead.